spec/registrant_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+
+  RSpec.describe Registrant do
+    before(:each) do
+      @registrant_1 = Registrant.new('Bruce', 18, true )
+      @registrant_2 = Registrant.new('Penny', 15 )
+      @registrant_3 = Registrant.new('Tucker', 15 )
+
+      @facility_1 = Facility.new(
```

Comment:
See my comment above about the more conventional way to put this on separate lines.